### PR TITLE
Properly set default fiat value for non-USD amounts

### PR DESCRIPTION
**Pull request by paullinator**
Convert $500 to fiat denom of wallet and round to highest decimal precision. This prevents getting a default quote for excessively small or large amounts

### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [ ] Tested on Android device
- [x] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207000283682116

### CODE REVIEW

File: ios/edge.xcodeproj/project.pbxproj
Comment (Jon-edge) [inline ios/edge.xcodeproj/project.pbxproj:1]: What does this have to do with this change?

**Review by Jon-edge — CHANGES_REQUESTED** [review]

**Review by Jon-edge — APPROVED** [review]
I assume the pbxproj commit was accidental, approved if so

File: ios/edge.xcodeproj/project.pbxproj
Comment (paullinator) [reply ios/edge.xcodeproj/project.pbxproj:1]: accidental but I think it may be needed since the changes were automatically created upon compiling the app. Might have been a missing change that got left out of the RN 73 upgrade. If you see this change as well, then it's needed. Otherwise I'll drop it and call it a fluke.
